library/lua/gui/widgets.lua

```diff
@@ -391,6 +391,18 @@ function render_text(obj,dc,x0,y0,pen,dpen,disabled)
     obj.text_width = width
 end
 
+function render_scroll_icons(obj,dc)
+    if obj:getTextHeight() > obj.frame_body.height then
+        if obj.start_line_num ~= 1 then
+            dc:seek(dc.width-1, 0):string(obj.up_arrow_icon, obj.scroll_icon_pen)
```

Comment:
We want to set `frame_inset.r` to at least `1` to have an empty column, but the painter object `dc` can't draw there: after `dc:seek(dc.width, 0)` the `dc:isValidPos()` is `false`.